### PR TITLE
Mark features with one engine support 'experimental'

### PR DESCRIPTION
**Pull request by OnkarRuikar**
Similar to https://github.com/mdn/browser-compat-data/pull/17357

### Summary
These features are supported by only one engine, so should be marked as 'experimental'.

### Supporting details
https://github.com/mdn/content/pull/19569#pullrequestreview-1072232578

cc/ @teoli2003

### CODE REVIEW

**Review by teoli2003 — APPROVED** [review]
This looks ok to me. As there will be others PRs like this, I would like a second check on the first one.

@OnkarRuikar told me there are about 130 errors in mdn/browser-compat-data.

**Comment by teoli2003** [comment]
(Note we are synchronizing, with a script, what is displayed on MDN and what is in bcd. One day it will be displayed automatically but we had 100s of errors so it was really needed).

**Comment by queengooborg** [comment]
On the BCD side, we typically say that a feature that has been in one browser for a long period of time should be considered non-experimental, but what do you guys think about that?

**Comment by queengooborg** [comment]
> I used dialed down version the [test-status.ts#checkExperimental()](https://github.com/mdn/browser-compat-data/blob/75b2c77d0623c54b965b4598030b80269f4f2c2c/test/linter/test-status.ts#L18) to find these. If we consider flags, prefixes, and alternative names then there will be ~316 files modified in the entire repo.

The function isn't a fully-developed function to decide experimental status -- it's actually only used to set `experimental` to `false`, but not to `true`.  My goal is to automate both experimental and standard track status, so that contributors do not have to manually update them!

We have the full breakdown of when to set `experimental` to `true` or `false` in the data guidelines here: https://github.com/mdn/browser-compat-data/blob/main/docs/data-guidelines.md#choosing-an-experimental-status

**Comment by OnkarRuikar** [comment]
> The function isn't a fully-developed function to decide experimental status -- it's actually only used to set experimental to false, but not to true. My goal is to automate both experimental and standard track status, so that contributors do not have to manually update them!

We'll have involve git history in the script. 🤔

> We have the full breakdown of when to set experimental to true or false in the data guidelines here...

If you guys terminate this PR, I'll update the statuses on `mdn/content` side.
Looks like the special cases related to features behind flags and prefixes are going to be very few.

**Comment by teoli2003** [comment]
> On the BCD side, we typically say that a feature that has been in one browser for a long period of time should be considered non-experimental, but what do you guys think about that?

I think this is a bit problematic. If a browser develops an obscure API that no other vendors is interested in, it will just sit there forever and become non-experimental.

Also, often, for APIs that are the brainchild of one vendor, the first implementation is made by the spec writers. We all know how it goes when there is some internal pressure to release: if the feature works for the purpose, management pushes for its release. When a second vendor implements it later, that's when the actual review of the API happens and where changes are implemented (if still possible). If not possible, a better API is developed and adopted by all vendors, the original API staying in one browser, deprecated, until most users have migrated (or died out).

That's why I think that not marking such APIs as experimental is detrimental to web developers. The risk of change, or event marked as deprecated, is real. When not _experimental_ the web developer assumes that it is safe to consider it stable enough to stay forever on the Web platform. Features implemented by one vendor don't provide this guarantee.

**Comment by OnkarRuikar** [comment]
> If not possible, a better API is developed and adopted by all vendors, the original API staying in one browser, deprecated, until most users have migrated (or died out).
>
> …Features implemented by one vendor don't provide this guarantee.

I think the best example of this could be: https://bugs.chromium.org/p/chromium/issues/detail?id=695508#c18

File: api/_globals/createImageBitmap.json [hunk 1]
```diff
@@ -247,7 +247,7 @@
             "webview_android": "mirror"
           },
           "status": {
-            "experimental": false,
```
Comment (Elchi3) [inline api/_globals/createImageBitmap.json:250]: This is actually in Safari, not sure which version though.
https://github.com/WebKit/WebKit/blob/main/Source/WebCore/html/ImageBitmapOptions.idl#L37

But that makes it non-experimental.

File: api/CSSPropertyRule.json [hunk 2]
```diff
@@ -28,7 +28,7 @@
           "webview_android": "mirror"
         },
         "status": {
-          "experimental": false,
+          "experimental": true,
```
Comment (Elchi3) [inline api/CSSPropertyRule.json:31]: Also in Safari https://github.com/WebKit/WebKit/blob/main/Source/WebCore/css/CSSPropertyRule.idl

File: api/CSSFontFeatureValuesRule.json [hunk 3]
```diff
@@ -26,7 +26,7 @@
           "webview_android": "mirror"
         },
         "status": {
-          "experimental": false,
```
Comment (Elchi3) [inline api/CSSFontFeatureValuesRule.json:29]: In safari actually https://github.com/WebKit/WebKit/blob/main/Source/WebCore/css/CSSFontFeatureValuesRule.idl

File: api/BaseAudioContext.json [hunk 4]
```diff
@@ -699,7 +699,7 @@
               "webview_android": "mirror"
             },
             "status": {
-              "experimental": false,
```
Comment (Elchi3) [inline api/BaseAudioContext.json:702]: Supported in Firefox https://searchfox.org/mozilla-central/source/dom/webidl/PeriodicWave.webidl#14

**Review by Elchi3 — CHANGES_REQUESTED** [review]
Quite a lot of support data is wrong here thus leading to experimental setting. I think we should rather fix the data than set experimental.

**Pull request closed without merging**